velox/exec/WindowPartition.cpp

```diff
@@ -364,7 +364,7 @@ void WindowPartition::updateKRangeFrameBounds(
   vector_size_t start = 0;
   vector_size_t end;
   RowColumn frameRowColumn = columns_[frameColumn];
-  RowColumn orderByRowColumn = columns_[inputMapping_[orderByColumn]];
+  RowColumn orderByRowColumn = data_->columnAt(orderByColumn);
```

Comment:
nit: Could you add a comment above that frameColumn is column index into the original input rows, while orderByColumn is a column index into rows in data_ after column reordering? This would help avoid confusion in the future.